### ISSUE TITLE
[Suggestion] Effect: Boundary Break/Airbrake/Noclip

### ISSUE THREAD

**Issue by TheRealAyCe** [gta-chaos-mod/ChaosModV#243]
Temporarily turn off collision for the player, so they can fly anywhere they want.

Or maybe leave the player character where they are and control the camera only, flying away and leaving your body behind.

**Comment by OHSHIET** [comment]
I had somewhat the same idea, I remember writing about it in Discord, but it was about only going through walls, not flying.

**Comment by kolyaventuri** [comment]
Yeah it would need to be only no-clip in the X / Y direction. No-clip in the Z axis would cause you to clip through the floor, and any controls for "flying" may not be intuitive enough in the effect time? Though I suppose you could do it based off of the camera view or mimic flight controls? Just don't want the user immediately falling through the world, primarily
